docs/zeebe/dynamic_scaling.md

```diff
@@ -0,0 +1,209 @@
+# Dynamic Scaling
+
+*Dynamic Scaling* refers to the process of increasing the number of partitions of a zeebe cluster once a cluster has been created.
+Before this feature, the number of partitions could not be changed.
+
+> [!NOTE]
+> Dynamic scaling is currently a **beta feature**. The redistribution phase is fully implemented and stable, while the relocation phase (subscription migration) is not yet implemented and is planned for future releases.
+
+The scaling process is comprised of two sequential phases that ensure new partitions are fully integrated into the cluster:
+
+1. **Redistribution**: Deploy common resources (process definitions, decisions, forms) to new partitions so they can process events
+2. **Relocation**: Redistribute message/signal subscriptions across all partitions (old and new) for even performance when correlating messages *(currently not implemented)*
+
+## Redistribution
+
+The redistribution phase ensures that new partitions receive all the necessary global resources (process definitions, decisions, forms) required to process workflow instances. This phase involves taking a snapshot of global state from partition 1 and distributing it to newly created partitions.
+
+### What Gets Redistributed
+
+During redistribution, the following resources are automatically deployed to new partitions:
+- **Process Definitions**: All deployed BPMN processes
+- **Decision Definitions**: DMN decision tables and DRD definitions  
+- **Form Definitions**: forms linked to user tasks
+- **Global Configuration**: Routing state and partition configuration
+
+**Important**: Partition-specific state (active process instances, jobs, message subscriptions) remains on the original partitions and is **not** redistributed.
+
+A sequence diagram of the redistribution process can be found below:
```

Comment:
This diagram is not just re-distribution, but covers the whole scaling right?

Reply:
This is just for redistribution, no relocation is included in the diagram